rpms.in.yaml

```diff
@@ -0,0 +1,8 @@
+contentOrigin:
+  repofiles:
+    - ubi.repo
+packages:
+  - unzip
+  - jq
+  - wget
```

Comment:
These can be removed and the rpm lockfiles refreshed, as they are no longer needed in the Dockerfile.